data/json/recipes/recipe_electronics.json

```diff
@@ -300,6 +300,38 @@
   ]
 },{
   "type" : "recipe",
+  "result": "sheet_metal_lit",
+  "category": "CC_ELECTRONIC",
+  "subcategory": "CSC_ELECTRONIC_LIGHTING",
+  "skill_used": "electronics",
+  "difficulty": 1,
+  "time": 15000,
+  "reversible": false,
+  "autolearn": false,
+  "book_learn": [[ "manual_electronics", 1 ] , [ "mag_electronics", 1 ]],
+  "tools": [
+    [
+      [ "soldering_iron", 5 ],
+      [ "toolset", 5 ]
+    ]
+  ],
+  "components": [
+    [
+      [ "sheet_metal", 1 ]
+    ],
+    [
+      [ "lightstrip_inactive", 1 ],
```

Comment:
If I may speak my train of thought...
This had occurred to me too, but the source of fuel for the light strip is typically batteries, something that has no on/off circuit logic on its own. However, the source power can have an on/off switch. This switch does not exist on the lightstrip, that is merely the "wiring" for the sheet metal, the strip and wires to provide light if provided with electricity. The current can be toggled at the controls.

ANYWAY... if you want me to change it I can. :-)
Anything for you @KA101